### PR TITLE
Stop using aws s3 built in paginator due to errors, handle pagination…

### PR DESCRIPTION
**Pull request by bbalser**
… with streams::unfold

### CODE REVIEW

File: file_store/src/file_store.rs [hunk 1]
```diff
@@ -72,34 +72,53 @@ impl FileStore {
...
+                    .into_iter()
+                    .filter_map(|obj| {
+                        if FileInfo::matches(obj.key().unwrap_or_default()) {
+                            Some(FileInfo::try_from(&obj).unwrap())
+                        } else {
+                            None
+                        }
+                    })
+                    .filter(move |info| after.map_or(true, |v| info.timestamp > v))
+                    .filter(move |info| before.map_or(true, |v| info.timestamp <= v))
```
Comment (andymck) [inline file_store/src/file_store.rs:115]: for this to be a 'true' before we should drop the equal sign, in the same way as the 'after' above

**Review by andymck — APPROVED** [review]
lgtm and has been running on mainnet verifier for past two days

File: file_store/src/file_store.rs (hunk 1 above)
Comment (bbalser) [reply file_store/src/file_store.rs:115]: So you would rather have `before` be exclusive?  I think we need one of them to be inclusive and given how `start_after` works in the s3api, I think it makes sense that before is inclusive.

Comment (bbalser) [reply file_store/src/file_store.rs:115]: @madninja you okay with this?

**Review by madninja — APPROVED** [review]

**Pull request merged**